spec/models/conference_spec.rb

```diff
@@ -41,6 +41,16 @@
       end
     end
   end
+  
+  describe '#dates' do
+    subject { FactoryGirl.build_stubbed(:conference) }
+    
+    it 'has a date range' do
```

Comment:
I tried everything I could think of, but it wouldn't pass. (I used the correct keyword `end_date`.)
It boils down to that after the `.with`-part, `subject.dates` is nil. 

Later, I changed the `dates` method to `date_range` to include compact, and then of course the method does not return a DateRange anymore, but a formatted string. 

Help? I think it makes sense to check if the outgoing message is being sent - if I understand the Sandi Metz rules correctly. But how?